### DISCUSSION TITLE
How to mention Author in Slack via Argo CD Notifications?

### DISCUSSION THREAD

**Discussion by siansiansu** [argoproj/argo-cd#10573] [Q&A]
I've set up the Argo CD Notifications, but I have no idea how to mention commit author in the Slack message.
Here is my part of the setup for tagging author

```json
{
    "type": "mrkdwn",
    "text": "*Author:*\n<@{{(call .repo.GetCommitMetadata .app.status.sync.revision).Author}}>"
},
```

As you see, it's always shows the text only.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/33391637/189721015-a0a26348-caae-4b6f-ad9e-174e0f8d70e9.png">

Is any a way to mention person in the Slack?

**Comment by naviat** [comment]
@siansiansu do you have any updates on a solution for this issue?

**Comment by mkantzer** [comment]
This would need to be built into the notification engine itself, most likely.

I have submitted a [feature request](https://github.com/argoproj/notifications-engine/issues/230) to add this functionality.
